data_reports/116_Jorgensen_YKDE.Rmd

```diff
@@ -1,30 +1,205 @@
----
-title: "116_Jorgensen_YKDE"
-author: "Julie Bielecki"
-date: "3/23/2021"
-output: html_document
----
+# Jorgensen YKDE
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+The Jorgensen YKDE data set in ISCN3 contains `r nrow(dataset_layer)` layer-level rows and `r nrow(dataset_profile)` profile-level information rows after cleaning for ISCN3.5.
```

Comment:
Need to actually put the number here since the dataset_layer and dataset_profile are not ingested yet.